discord/abc.py

```diff
@@ -27,5 +27,58 @@
 Autogenerated by aliasgen.py
 """
 
-from nextcord.abc import AllowedMentions, Any, Callable, ChannelType, ClientException, Connectable, Dict, File, GCH, GuildChannel, GuildSticker, HistoryIterator, InvalidArgument, Invite, List, MISSING, Messageable, Optional, PermissionOverwrite, Permissions, PrivateChannel, Protocol, Role, Sequence, Snowflake, StickerItem, T, TYPE_CHECKING, Tuple, TypeVar, Typing, Union, User, VoiceClient, VoiceProtocol, _Overwrites, _Undefined, _undefined, annotations, asyncio, copy, overload, runtime_checkable, utils
-__all__ = ("Snowflake", "User", "PrivateChannel", "GuildChannel", "Messageable", "Connectable")
\ No newline at end of file
+from nextcord.abc import (
```

Comment:
aliasgen doesn't generate linted scripts? Maybe add that as a step?

Reply:
hm?